### ISSUE TITLE
PVR Recordings - drive size and upcomming recording.

### ISSUE THREAD

**Issue by loggio** [jurialmunkey/skin.aura#6]
Thought I'd put this here as I think you may have forgotten my request on the forums lol.
Can we get the drive size/remaining space on Recordings window. Seen here (bottom left):

![kodi_20180406_073130](https://user-images.githubusercontent.com/16231443/38393013-d6601312-396c-11e8-81fa-64a00015204e.png)

Also, nowhere in Aura does it tell you the "upcoming recording" as can be seen here (bottom left):
![kodi_20180406_073203](https://user-images.githubusercontent.com/16231443/38393068-ff294ade-396c-11e8-8164-3ca7109a9c02.png)

I understand these are part of the footer furniture overlay that does not exist in Aura and you may be having a hard time finding an appropriate place for this info.

Perhaps this info would be better shown at the top of the screen underneath the library location header that tells you where you are. Where you'd normally but library information etc. All you'd have to do is move the title and plot info down a little bit and add a small info line underneath the header that shows this sort of info when relevant.

**Comment by jurialmunkey** [comment]
Yeah it is mostly that I haven't found an appropriate place yet. I'm not a fan of having things move around. I like to keep consistent locations so that the eyes don't have to do a lot of scanning. I will try to find something for this tomorrow.

**Comment by loggio** [comment]
Great! No rush. 

Cheers.
